bundle/manifests/opendatahub-operator-webhook-service_v1_service.yaml

```diff
@@ -3,14 +3,11 @@ kind: Service
 metadata:
   annotations:
     service.beta.openshift.io/inject-cabundle: "true"
-    service.beta.openshift.io/serving-cert-secret-name: opendatahub-operator-controller-webhook-cert
+    service.beta.openshift.io/serving-cert-secret-name: opendatahub-operator-webhook-cert
```

Comment:
@ykaliuta Can you add why this rename is required?

Reply:
> @ykaliuta Can you add why this rename is required?

I have the same question :) https://github.com/opendatahub-io/opendatahub-operator/pull/1190#discussion_r1726897416

Reply:
;) No problem